src/components/port/CustomPortModel.ts

```diff
@@ -365,6 +366,26 @@ export  class CustomPortModel extends DefaultPortModel  {
 
     }
 
+    getTargetPorts = () => {
```

Comment:
Neat, these functions could cut down on some of the repetition I've got in the `addHover` and `removeHover` functions, as well as the ones in the copy & paste support.

Reply:
For this one, I've noticed that the `selectAllRelevantNodes` also highlights the links, and the addHover - removeHover accesses the properties through their names.